agent/xds/delta_test.go

```diff
@@ -1057,6 +1058,15 @@ func TestServer_DeltaAggregatedResources_v3_ACLEnforcement(t *testing.T) {
 
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
+			var stopped bool
+			var lock sync.RWMutex
```

Comment:
should we use a pointer here since we don't want copies of the mutex floating around?